tests/_support/Page/User.php

```diff
@@ -248,4 +248,19 @@ public function iApproveTheNewUser()
         $I->click("//td[text()='{$username}']/..//a[text()='Approve']");
     }
 
+    /**
+     * @Then /^I verify an email was sent$/
+     */
+    public function IVerifyEmailWasSent()
+    {
+        $I = $this->I;
+
+        $I->fetchEmails();
```

Comment:
There should probably be a check to see if an email server is configured and the feature turned on, and if not then skip the test.

Reply:
I added an env var USE_MAIL_FEATURE so that we can control functionality throughout the app depending if mail functionality is wanted or not.